server/src/main/java/com/tencent/rss/server/ShuffleServer.java

```diff
@@ -156,7 +160,14 @@ private void initialization() throws Exception {
 
     registerHeartBeat = new RegisterHeartBeat(this);
     shuffleFlushManager = new ShuffleFlushManager(shuffleServerConf, id, this, multiStorageManager);
-    shuffleBufferManager = new ShuffleBufferManager(shuffleServerConf, shuffleFlushManager);
+    memoryShuffleEnabled = shuffleServerConf.getBoolean(
+        ShuffleServerConf.SERVER_MEMORY_SHUFFLE_ENABLED);
+    if (memoryShuffleEnabled) {
+      shuffleBufferManager = new MemoryFSShuffleBufferManager(
+          shuffleServerConf, shuffleFlushManager);
+    } else {
+      shuffleBufferManager = new FSShuffleBufferManager(shuffleServerConf, shuffleFlushManager);
```

Comment:
We can unify FSShuffleBufferManaer and MemoryFSShuffleBufferManager.

Reply:
ok, keep only BufferManager to support both Local and Memory_Local